### PR TITLE
code clean-up

### PR DESCRIPTION
**Pull request by joshuaobinna**

### CODE REVIEW

File: src/components/productList/components/Product.js [hunk 1]
```diff
@@ -36,7 +32,7 @@ class Product extends React.Component {
         );
         const actions = [
             <Icon key={SHOPPING} type={SHOPPING} />,
-            <Icon key={INFO_CIRCLE} type={INFO_CIRCLE} onClick={this.showModal} />,
+            <Icon key={INFO_CIRCLE} type={INFO_CIRCLE} onClick={() => this.toggleModal(true)} />,
```
Comment (enyeInc) [inline src/components/productList/components/Product.js]: leave the this.showModal Logic the way it was previously

File: src/components/productList/components/Product.js [hunk 2]
```diff
@@ -55,7 +51,7 @@ class Product extends React.Component {
                 </Card>
                 <ProductModal
                     data={this.props.data}
-                    handleCancel={this.handleCancel}
+                    handleCancel={() => this.toggleModal(false)}
```
Comment (enyeInc) [inline src/components/productList/components/Product.js]: undo this please

File: src/components/productList/components/Product.js [hunk 3]
```diff
@@ -17,12 +17,8 @@ const {
...
     state = { visible: false }
 
-    showModal = () => {
-        this.setState({ visible: true });
-    }
-
-    handleCancel = () => {
-        this.setState({ visible: false });
+    toggleModal= bool => {
+        this.setState({ visible: bool });
```
Comment (enyeInc) [inline src/components/productList/components/Product.js]: leave this logic the way that it is

**Comment by Way2nnadi** [comment]
@joshuaobinna This PR doesn't do anything, unless you are going to add more code to it, i say close it

**Pull request closed without merging**